docs/pages/server-access/guides/ec2-discovery.mdx

```diff
@@ -288,30 +301,37 @@ spec:
     echo Teleport-{{ .MajorVersion }}
 version: v1
 ```
-Which, when retrieved by the Systems Manager document will evaluate to a script with the following contents:
+
+Which, when retrieved by the Systems Manager document will evaluate to a script
+with the following contents:
+
 ```sh
 echo teleport.example.com
 echo Teleport-(=teleport.version=)
 ```
 
 The default installer will take the following actions:
+
 - Add an official Teleport repository to supported Linux distributions.
 - Install Teleport via `apt` or `yum`.
 - Generate the Teleport config file and write it to `/etc/teleport.yaml`.
 - Enable and start the Teleport service.
 
 ## Troubleshooting
 
-If Installs are showing failed or instances are failing to appear check the Command history in AWS System Manager -> Node Management -> Run Command.  Select the instance-id of the Target to review Errors.  
+If Installs are showing failed or instances are failing to appear check the
+Command history in AWS System Manager -> Node Management -> Run Command.
+Select the instance-id of the Target to review Errors.
 
 ## Next Steps
```

Comment:
```suggestion
## Next steps
```
While we're editing this guide, I've suggested making this heading sentence case per the style guide.